Source/CarthageKitTests/ProjectSpec.swift

```diff
@@ -16,43 +16,69 @@ import Tentacle
 class ProjectSpec: QuickSpec {
 	override func spec() {
 		let directoryURL = NSBundle(forClass: self.dynamicType).URLForResource("CartfilePrivateOnly", withExtension: nil)!
```

Comment:
Looks like this should be moved into `it("should load a combined Cartfile when only a Cartfile.private is present")` now.